syntax = "proto3";
package HbData;

service Greeter {
  rpc Call (HbRequest) returns (HbReply) {}
}

message HbRequest {}

message HbReply {
  int32 return_code = 1;
  string ack = 2;
}

